src/app/components/OrderInput.tsx

```diff
@@ -805,10 +872,163 @@ function InputTooltip({ message }: { message: string }) {
 }
 
 // TODO(dcts): implement percentage slider in future PR
-function PercentageSlider() {
-  return <></>;
+interface PercentageSliderProps {
+  initialPercentage: number;
+  callbackOnPercentageUpdate: (newPercentage: number) => void;
+  isLimitOrder: boolean;
+  isBuyOrder: boolean;
+  isSellOrder: boolean;
```

Comment:
Should this be
isMarketBuyOrder
isMarketSellOrder
?

Or are we using those booleans also for LIMIT orders?

Reply:
@saidam90 can you comment on this?